test/test_browser.py

```diff
@@ -3069,13 +3097,14 @@ def test_sdl2_threads(self):
     self.btest_exit('test_sdl2_threads.c', args=['-pthread', '-sUSE_SDL=2', '-sPROXY_TO_PTHREAD'])
 
   @requires_graphics_hardware
+  @also_with_proxying
   @parameterized({
-    '': ([], True),
+    '': ([],),
     # fails on proxy
-    'closure': (['--closure=1', '-g1'], False),
+    'closure': (['--closure=1', '-g1'],),
   })
-  def test_sdl2_glshader(self, args, also_proxied):
-    self.reftest('test_sdl2_glshader.c', 'test_sdl_glshader.png', args=args + ['-sUSE_SDL=2', '-O2', '-sLEGACY_GL_EMULATION'], also_proxied=also_proxied)
+  def test_sdl2_glshader(self, args):
+    self.reftest('test_sdl2_glshader.c', 'test_sdl_glshader.png', args=args + ['-sUSE_SDL=2', '-O2', '-sLEGACY_GL_EMULATION'])
```

Comment:
This test will double in execution time. Do you have a sense if it's worth it?

Reply:
Not sure, I'm happy to revert back.

Reply:
Actually as the comment above states, closure doesn't work with proxying (or at least it generates warnings)